### ISSUE TITLE
Not sure if the code is public domain

### ISSUE THREAD

**Issue by ioistired** [kspalaiologos/malbolge-lisp#1]
There's a CC-0 dedication in the file, but the interpreter itself outputs:
```
MalbolgeLisp. Copyleft (C) 2020 THE GREEK EMPRESS.
All wrongs reserved.
```
If it is indeed copyleft, which license is it under? If it's not, can this greeting be changed?

**Comment by iczelia** [comment]
I'll change the banner later. The interpreter is, indeed, CC0. If you want to redistribute it, feel free to ignore the message and treat the program as CC0'd.

**Comment by iczelia** [comment]
solved as of v1.1

...lol
